airgun/entities/oscapreport.py

```diff
@@ -0,0 +1,79 @@
+from wait_for import wait_for
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.utils import retry_navigation
+from airgun.views.oscapreport import (
+    RemediateModal,
+    SCAPReportDetailsView,
+    SCAPReportView,
+)
+
+
+class OSCAPReportEntity(BaseEntity):
+    endpoint_path = '/compliance/arf_reports'
+
+    def search(self, search_string):
+        """Search for SCAP Report
+
+        :param search_string: how to find the SCAP Report
+        :return: result of the SCAP Report search
+        """
+        view = self.navigate_to(self, 'All')
+        return view.search(search_string)
+
+    def details(self, search_string, widget_names=None, limit=None):
+        """Read the content from corresponding SCAP Report dashboard,
+            clicking on the link in Reported At column of
+            SCAP Report list
+
+        :param search_string:
```

Comment:
what is this search string ? is this any type of entity?

Reply:
Unfortunately a link to the correct arf report is in the column `Reported At` and you get there by clicking the link saying e.g. `1 day ago`... so search actually searches by a (not visible) id in the link.
You call it like this:
```
session.oscapreport.remediate(f'id={arf_id}', title)
```